src/set_theory/game/basic.lean

```diff
@@ -146,7 +146,7 @@ but to prove their properties we need to use the abelian group structure of game
 Hence we define them here. -/
 
 /-- The product of `x = {xL | xR}` and `y = {yL | yR}` is
-`{xL*y + x*yL - xL*yL, xR*y + x*yR - xR*yR | xL*y + x*yR - xL*yR, x*yL + xR*y - xR*yL }`. -/
+`{xL*y + x*yL - xL*yL, xR*y + x*yR - xR*yR | xL*y + x*yR - xL*yR, xR*y + x*yL - xR*yL}`. -/
```

Comment:
Is there a rationale behind this change? If it's just for aesthetic purposes, I'd rather not touch it. Might screw over the surreal multiplication proof (which I really need to get back to...)

Reply:
The definition was not defeq to what was in docstring.

Reply:
Ah, I didn't realize only the docstring was changed.